user_sessions/migrations/0004_session_created.py

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import datetime
+
+def set_created(apps, schema_editor):
+    Session = apps.get_model('user_sessions', 'Session')
+
+    for obj in Session.objects.all():
+        obj.created = obj.last_activity
+        obj.save()
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('user_sessions', '0003_auto_20161205_1516'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='session',
+            name='created',
+            field=models.DateTimeField(default=datetime.datetime(2017, 4, 18, 12, 11, 0, 286186), auto_now_add=True),
+            preserve_default=False,
+        ),
+        migrations.RunPython(set_created),
```

Comment:
Migrations should be reversible, so please also provide the `reverse_code` argument here. It doesn't need to do anything; see also this snippet from the Django docs:

> Pass the RunPython.noop method to code or reverse_code when you want the operation not to do anything in the given direction. This is especially useful in making the operation reversible.